docs/arch/adr-20.md

```diff
@@ -0,0 +1,36 @@
+# ADR 20: Transcription Task
+
+Created: November 12, 2019
+
+## Context
+
+ILMS funded us to experiment with transcription tools. The project [Anti-Slavery Manuscripts](https://www.antislaverymanuscripts.org/classify) has a workflow where volunteers transcribe lines of text first by marking the line, then adding the transcription in a sub-task. This task is slightly unique in that:
+
+- Incomplete classifications can be submitted
+- Previous transcriptions from caesar are loaded and presented as an option in the sub-task
+  - volunteers can select the previous transcription, edit it, and submit a new transcription
+  - lines with previous transcriptions or retired lines are displayed visually by color
+
+## Decision
+
+We will be porting the ASM functionality to the main classifier as a new transcription task. The task will be composite of:
+
+- A drawing line task that is created by two pointer down and up events to mark the points of the line.
+  - the starting point and ending point will have visual indicators.
+  - Note that ASM leveraged a hacked polygon tool to do this, but we should have a line tool variant specifically for this instead. Downstream aggregation can leverage polygon type aggregations if it fits.
```

Comment:
We had to abide by the polygon tool because that was the only tool supported through the aggregator at the time (2017). A line tool aggregator should now exist.

Reply:
https://github.com/zooniverse/aggregation-for-caesar